salt/metalk8s/kubernetes/etcd/prepared.sls

```diff
@@ -0,0 +1,9 @@
+{%- from "metalk8s/repo/macro.sls" import build_image_name with context %}
+
+{%- set image = build_image_name("etcd") -%}
```

Comment:
Do we really need to set this jinja var ? Putting directly `build_image_name("etcd")` in `name` of `containerd.image_managed` is simpler IMO